resources/translations/cs_CZ/cooling/cool_min_speed.md

```diff
@@ -2,7 +2,7 @@ Minimální rychlost
 ====
 Pokud tisk vrstvy zabírá velmi málo času, rychlost tisku se sníží, takže vrstva bude stále k tisku potřebovat [minimální čas vrstvy](cool_min_layer_time.md). Nebude však snížena pod tuto minimální rychlost. Pokud tisk vrstvy minimální rychlostí trvá kratší dobu, než je minimální čas vrstvy, tiskárna počká na konci vrstvy, dokud neuplyne minimální čas vrstvy.
 
-Je důležité respektovat minimální dobu vrstvy, aby předchozí vrstva mohla ztuhnout, než položí na vrch další vrstvul. Snížení rychlosti tisku je užitečné, protože ventilátory tiskové hlavy mohou na vlákno déle foukat. Na druhé straně je nadměrné snížení rychlosti tisku škodlivé, protože teplo trysky se může šířit do spodních vrstev a na sousední stěny. To může způsobit, že povrch bude velmi špinavý a způsobí místní ochabnutí. Proto existuje také minimální rychlost. Na konci vrstvy se tisková hlava normálně pohybuje trochu nahoru (pokud není deaktivován parametr [Zvednout hlavu](cool_lift_head.md)), který umožňuje ventilátoru foukat na vlákno o něco déle, dokud neuplyne minimální doba vrstvy. Tiskárna poté přejde na další vrstvu.
+Je důležité respektovat minimální dobu vrstvy, aby předchozí vrstva mohla ztuhnout, než položí na vrch další vrstvu. Snížení rychlosti tisku je užitečné, protože ventilátory tiskové hlavy mohou na vlákno déle foukat. Na druhé straně je nadměrné snížení rychlosti tisku škodlivé, protože teplo trysky se může šířit do spodních vrstev a na sousední stěny. To může způsobit, že povrch bude velmi špinavý a způsobí místní ochabnutí. Proto existuje také minimální rychlost. Na konci vrstvy se tisková hlava normálně pohybuje trochu nahoru (pokud není deaktivován parametr [Zvednout hlavu](cool_lift_head.md), který umožňuje ventilátoru foukat na vlákno o něco déle, dokud neuplyne minimální doba vrstvy. Tiskárna poté přejde na další vrstvu.
```

Comment:
Similarly, you removed a closing bracket here that I think you didn't intend to remove. The bracketed text starts at the word `pokud`.

Reply:
fixed